docs/Guides/Scim/Guide-to-sync-users-in-vendasta.md

```diff
@@ -0,0 +1,54 @@
+# Guide to use Vendasta SCIM APIs to sync Users
+
+
+## Overview
+System for Cross-domain Identity Management ([SCIM](https://en.wikipedia.org/wiki/System_for_Cross-domain_Identity_Management)) focuses on syncing user accounts and permissions between systems but has an extension system that allows syncing any type of record. There are several well known endpoint paths for discovering what the server supports and resource schema definitions.
+
+This guide provides the information of
+
+- To create a service account in vendasta and authorization token to access vendasta APIs
+- List of Vendasta SCIM APIs and the way to use it.
+
+
+## Step 1 : Pre-requisites for accessing the Vendasta's SCIM APIs
+
+### 1. Namespace
+
+This must be the id of the partner that wish to manage users in sync with Vendasta. It should be unique for a partner.
+
+The users access Partner Center is that when a new Channel Partner signs up a new Partner is created and this generates a PID. The PID is then set as the namespace of all the SCIM Users. 
```

Comment:
We should adjust the wording to be in alignment with Vendasta's [voice and tone guidelines](https://vendasta.jira.com/wiki/spaces/GLXY/pages/686292995/Voice+and+tone). I dropped more related links in Slack.

In particular for this guide I want you to imagine that you are on a call with or writing an email to a developer that works for one of our partners. Instead of calling them "the users" or "a partner with Vendasta" it is ok to use terms like "you" or "your company". 

The documentation that is generated based on the scim.yaml file provides the details of what is supported. The guides are meant to provide step by step directions to solve a job to be done. They usually start with a description of the job and a high level overview of the solution that a non-technical person can understand. They then go into technical details.

If the person we are having a conversation with said "I have a database of users that I want to keep in sync with Vendasta, what is the easiest way to do that?" what would your email response be? 

I think it would start out by saying that "if you are using Azure Active Directory, Okta, OneLogin, Oracle Identity Cloud Service, SailPoint or similar systems they have built in tools to synchronizing users using the SCIM protocol. If you would like to build your own integration following the SCIM protocol here are the API calls you will need to make."

The [Create a Business](https://developers.vendasta.com/platform/ZG9jOjEzMTU1Mjky-create-a-business-user) guide is not quite step by step but has some useful info.
I like the way that the [Request reviews from your customers](https://developers.vendasta.com/platform/ZG9jOjMyNTk1MTM4-request-a-review-for-a-business-location) guide includes example responses as part of the guide. 
Some of the newer guides include a link to the generated docs after each example API call. [Example](https://developers.vendasta.com/platform/ZG9jOjI4NzUxNDM0-getting-advertising-connected-accounts)